pkgs/by-name/ra/redbean/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc  ];
+  arch =
+    if stdenv.is64bit
+    then "x86_64"
+    else "x86";
+in
+stdenv.mkDerivation rec {
+  pname = "Redbean";
```

The `pname` must be lowercase
```suggestion
  pname = "redbean";
```